src/app/auth.service.spec.ts

```diff
@@ -0,0 +1,17 @@
+import {TestBed} from '@angular/core/testing';
+import {AuthService} from './auth.service';
+
+describe('AuthService', () => {
+  let service: AuthService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(AuthService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  /* TODO Test 1. firebaseGoogleAuth(), 2. firebaseSignOut() */
```

Comment:
don't merge until they have tests

(otherwise you'd possibly be merging broken code)

Reply:
(Unresolving - I don't see these changes, yet)